src/ssl/ServerBump.cc

```diff
@@ -62,13 +62,12 @@ Ssl::ServerBump::attachServerSession(const Security::SessionPointer &s)
     serverSession = s;
 }
 
-const Security::CertErrors *
+Security::CertErrors *
```

Comment:
This looks like it may be another source of leak risks. Can it become a `Pointer` reference yet?

Reply:
This is a stored errors _accessor_ method rather than errors sharing/extracting/making method. If errors were guaranteed to exist, then the method would just return a reference to CertErrors instead of a pointer. The pointer is (naturally) used to be able to indicate lack of errors.

I do not see a leak risk in this particular API, but if somebody were to change this method return type to avoid raw pointers, then the adjusted method would return something like `std::optional< std::reference_wrapper<CertErrors> >`. A raw pointer is a much simpler equivalent of that construct!